cmd/ocm-backplane/console/console.go

```diff
@@ -1172,29 +1169,26 @@ func (ce *dockerMac) runMonitorPlugin(containerName string, consoleContainerName
 // put a file in place for container to mount
 // filename should be name only, not a path
 func (ce *podmanLinux) putFileToMount(filename string, content []byte) error {
-	// for files in linux, we put them into the user's backplane config directory
-	configDirectory, err := config.GetConfigDirctory()
-	if err != nil {
-		return err
-	}
-	dstFileName := filepath.Join(configDirectory, filename)
+	// for files in linux, we put them into the tmp directory
+	tmpDir := os.TempDir()
```

Comment:
can we reuse the tmpDir in the other function?

Reply:
ok, seems it is not creating new dir in the tmp, then the question would be do we want to put the bp-cli related files into a separated directory under tempdir?

Reply:
Saving cluster specific nginx config into the `bp-cli-config` folder is not ideal. This will mainly block multi-cluster login since one console command writes cluster specific nginx conf to './config/backplane/nginx.conf' and other login override the same file. 

We should move files to `/tmp` or user defined folder and should rename the file with `{cluster-id}-nginx.config` that will reenable multi cluster login.

@feichashao regarding unit tests, if it's hard lets create a new card

Reply:
I mean we do not put it into the root of `/tmp`. But maybe something like `/tmp/backplane-cli/`, just to centralize the config files
If consider the cluster id as well, it may like `/tmp/backplane-cli/$CLUSTER_ID-nginx.conf`